apps/native-component-list/package.json

```diff
@@ -9,7 +9,6 @@
     "postinstall": "expo-yarn-workspaces postinstall",
     "build:web": "expo build:web --no-pwa",
     "web": "expo start:web --https",
-    "eject": "SDK_VERSION=43.0.0 expo eject",
```

Comment:
This needs to stay.

Reply:
Why's that? Won't that not work anymore if the command is gone (similar to the templates)?

Reply:
the command needs to stay and be `npx expo prebuild` since we use NCL for an informal E2E test for config plugins.